packages/common/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@web3modal/common",
+  "version": "3.2.0",
```

Comment:
I think there is a version miss-alignment here with other packages (3.2.1 is latest), this could be causing issues with tests as Sven suggested.